concreep-refilled/control.lua

```diff
@@ -144,7 +202,19 @@ function _reinit_reentrant_level1_start()
 	global.no_work_counter = 0
 
 	-- set up outer arrays to iterate
-	global.reinit_surfaces = array_shallowcopy(game.surfaces)
+	local listOfSurfaces = {}

```

Comment:
Think i have done that now - however not quite sure how to check now if we even have a single valid surface in case the user screws up the input - I guess I will find a use for the script.on_configuration_changed